File: springboot/src/main/java/com/example/crowdsourcing/dao/bean/OffineOrdering.java
package com.example.crowdsourcing.dao.bean;


import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import java.util.Date;

@Entity
public class OffineOrdering {


    @Id
    @GeneratedValue
    int id;

    @Column
    int peopleId;

    @Column
    int offineOrderId;

    @Column
    Date createDate;

    @Column
    Date finishDate;


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getPeopleId() {
        return peopleId;
    }

    public void setPeopleId(int peopleId) {
        this.peopleId = peopleId;
    }

    public int getOffineOrderId() {
        return offineOrderId;
    }

    public void setOffineOrderId(int offineOrderId) {
        this. offineOrderId = offineOrderId;
    }

    public Date getCreateDate() {
        return createDate;
    }

    public void setCreateDate(Date createDate) {
        this.createDate = createDate;
    }

    public Date getFinishDate() {
        return finishDate;
    }

    public void setFinishDate(Date finishDate) {
        this.finishDate = finishDate;
    }





}

File: flutter/crowdsourcing/lib/models/object/order/offine/OffineOrdering.dart
class OffineOrdering {
  int id;

  int peopleId;

  int offineOrderId;

  DateTime createDate;

  DateTime finishDate;

  static const String Id = "id";
  static const String PeopleId = 'peopleId';
  static const String OffineOrderId = 'offineOrderId';
  static const String CreateDate = 'createDate';
  static const String FinishDate = 'finishDate';

  OffineOrdering(
      {this.id,
      this.peopleId,
      this.offineOrderId,
      this.createDate,
      this.finishDate});

  static OffineOrdering fromJsonMap(Map<String, dynamic> map) {
    if (map == null) return null;
    return OffineOrdering(
        id: map[Id],
        peopleId: map[PeopleId],
        offineOrderId: map[OffineOrderId],
        createDate:  map[CreateDate]!=null? DateTime.fromMillisecondsSinceEpoch(map[CreateDate]):null,
        finishDate: map[FinishDate]!=null?DateTime.fromMillisecondsSinceEpoch(map[FinishDate]):null);
  }

  Map toJson() => {
        Id: id,
        PeopleId: peopleId,
        OffineOrderId: offineOrderId,
        CreateDate: createDate?.millisecondsSinceEpoch,
        FinishDate: finishDate?.millisecondsSinceEpoch
      };
}
